ext/drmgr/drmgr.c

```diff
@@ -1256,7 +1263,16 @@ drmgr_register_thread_init_event_ex(void (*func)(void *drcontext),
                                     drmgr_priority_t *priority)
 {
     return drmgr_generic_event_add(&cb_list_thread_init, thread_event_lock,
-                                   (void (*)(void)) func, priority);
+                                   (void (*)(void)) func, priority, NULL);
+}
+
+DR_EXPORT
+bool
+drmgr_register_thread_init_event_usr_data(void (*func)(void *drcontext, void *usr_data),
```

Comment:
I think including the e (i.e., ...event_user_data) is much nicer in the function name.  I would also vote for spelling it out everywhere else.  WDYT?